src/olympia/devhub/templates/devhub/new-landing/components/navigation.html

```diff
@@ -11,6 +11,9 @@
         <li class="show-on-desktop"><a href="{{ url('devhub.addons') }}">
           {{ _('My Add-ons') }}
         </a></li>
+        <li><a href="https://extensionworkshop.com/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=devhub-menu-link">
```

Comment:
If I am not wrong, we are going to show it everywhere so did not add `show on desktop` here.
Correct me if i am wrong.

Thank you

Reply:
I guess?
@caitmuenster do we want the link shown on mobile-sized views or just when viewed on a larger screen? (Not sure there's enough space to fit it on mobile tbh, unless we drop one of the other links)

Reply:
We can drop the link on mobile. :) Thanks for checking in!

Reply:
Thanks for clarification @caitmuenster!